### PR TITLE
Update for bionic release.

### PR DESCRIPTION
**Pull request by fabricematrat**

### CODE REVIEW

File: internal/series/series.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ var Series = map[string]SeriesInfo{
 	"yakkety": {true, Ubuntu, true, 1.103},
```
Comment (mhilton) [inline internal/series/series.go]: yakkety and zesty are no longer supported so should have their last two values changed to `false, 0`.

File: internal/series/series.go [hunk 2]
```diff
@@ -52,7 +52,7 @@ var Series = map[string]SeriesInfo{
...
 	"zesty":   {true, Ubuntu, true, 1.104},
 	"artful":  {true, Ubuntu, true, 1.105},
-	"bionic":  {true, Ubuntu, true, 0}, // TODO 1.15 when released
+	"bionic":  {true, Ubuntu, true, 1.15},
```
Comment (mitechie) [inline internal/series/series.go]: shouldn't this be 1.5 or something higher than 1.375?

**Review by mitechie — COMMENTED** [review]
question

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/charmstore/2180/

**Review by mhilton — APPROVED** [review]
LGTM (but with another quick change)

File: internal/series/series.go (hunk 2 above)
Comment (mhilton) [reply internal/series/series.go]: I think this looks right, 1.15 is higher than xenial's 1.1375.

Comment (mitechie) [reply internal/series/series.go]: My bad, I read that as 1.375 and missed the 1 in there. I retract my comment.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/charmstore/2181/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/charmstore/2182/

**Comment by fabricematrat** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/charmstore-merge

**Pull request merged**